### PR TITLE
ui: display Flask's built-in flash messages

### PR DESCRIPTION
**Pull request by mars-f**
Flask has a built-in support for displaying flash messages
for user feedback.  This patch adds the necessary components
to display those flash messages in the UI of all Lando pages.

### CODE REVIEW

**Comment by mars-f** [comment]
@purelogiq I need to update the CSS to use [SUIT CSS](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md) rules.

**Comment by mars-f** [comment]
@purelogiq I chose Bulma's [Modal card](https://bulma.io/documentation/components/modal/) component, but the Bulma [Message](https://bulma.io/documentation/components/message/) component might be better.  However I'm not sure where the flash message would be placed if I used it.  The Message component doesn't float over the screen like the Modal does.

Perhaps the Message component would be placed inline with the content at the top of the page, before the `<h1>`?

**Comment by mars-f** [comment]
Selection_067

Version using the Bulma notification component and styled to match the revision badges.

**Comment by mars-f** [comment]
Selection_068

Warning and Info messages using the Bulma default pallet and the Badge warning colour as a default.

**Comment by mars-f** [comment]
This is ready for review

**Comment by smacleod** [comment]
The stack page itself has something which displays very similarly, used for showing error messages:
https://github.com/mozilla-conduit/lando-ui/blob/7e92a65731d652c0f10f01e9cca843f84bb91679/landoui/templates/stack/stack.html#L6-L15 

Do flash messages not have an error option? I'm wondering what you're planning to use this for and if you're going to migrate the current functionality that displays in a similar way?

**Comment by mars-f** [comment]
> The stack page itself has something which displays very similarly, used for showing error messages:
> 
> https://github.com/mozilla-conduit/lando-ui/blob/7e92a65731d652c0f10f01e9cca843f84bb91679/landoui/templates/stack/stack.html#L6-L15
> 
> Do flash messages not have an error option? I'm wondering what you're planning to use this for and if you're going to migrate the current functionality that displays in a similar way?

Flash messages are persisted by Flask across requests so they work well for messaging after the user takes an action like form submission.  For example, in the form submission code you call `flash("Yay!", "success")` and the next page the user sees will have the message "Yay!" at the top in green.

I'm using flash messages in 3 places: as a success message after submitting a sec-review request; As a success message after updating a secure commit message; And as a warning at the top of the sec-approval page if the user has not supplied their API token:

Selection_070

Flash messages can have arbitrary categories.  I coded the two I need for sec-approval but we can add more easily.  I mapped the notification names to the [Bulma CSS Notification](https://bulma.io/documentation/elements/notification/) categories for simplicity and symmetry.

We could map the existing error message code in a similar way.  I have not looked at the code that generates the error messages but I expect it would be easy to port: call `flash(message, category)` and render the template.

File: landoui/assets_src/css/components/FlashMessages.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@import "../colors";
+
+.FlashMessages {
+  padding: 1.5rem;
+
+  &-content {
+    margin-bottom: 1rem;
```
Comment (purelogiq) [inline landoui/assets_src/css/components/FlashMessages.scss]: Nit pick: This combination really pushes the content of the page down a lot. I think we could reduce the amount of padding on the top and bottom and the bottom margin a bit. Up to you.

Comment (mars-f) [reply landoui/assets_src/css/components/FlashMessages.scss]: @purelogiq Done.
Screenshot_2019-12-17 D4 - Lando - Mozilla

Comment (purelogiq) [reply landoui/assets_src/css/components/FlashMessages.scss]: @mars-f I don't see any change between the screenshots or the code. Is that just me?

Comment (mars-f) [reply landoui/assets_src/css/components/FlashMessages.scss]: @purelogiq oops, I pushed the update to a branch with the same name on another remote.  This PR is fixed now, have a look again?

Comment (purelogiq) [reply landoui/assets_src/css/components/FlashMessages.scss]: LGTM!

**Pull request merged**